src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <img
+        class="product__img"
+        src="./images/imac.jpeg"
+        alt="product-img"
+      />
+      <div class="product__list">
+        <span class="name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </span>
+        <span class="code">Product code: 195434</span>
+        <div class="popular">
+          <div class="stars">
```

Comment:
Add the modifier class
```suggestion
          <div class="stars stars--4">
```